proto/ibc/applications/middleware/fee/v1/genesis.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package ibc.applications.middleware.fee.v1;
+
+import "gogoproto/gogo.proto";
+import "ibc/applications/middleware/fee/v1/fee.proto";
+import "ibc/core/channel/v1/channel.proto";
+option go_package = "github.com/cosmos/ibc-go/modules/apps/29-fee/types";
+
+
+// GenesisState defines the fee middleware genesis state
+message GenesisState {
+  map<string, ibc.applications.middleware.fee.v1.Fee> packet_fees = 1;
```

Comment:
I'm not sure it is safe to use maps since they aren't determinate and genesis files need to be ordered the same (for verification), there's usually a lot of concern about ensuring `ExportGenesis` is determinate 

I'm also fairly certain this is why the genesis file is filled with a ton of arrays and structs which do indirect mappings

Reply:
Ahhhh, of course, yeah makes sense! Same reason we don't use maps in smart contracts I suppose.

Reply:
I think a helper struct might be useful here, a struct which contains the packet id, and each of the fees. Then the genesis can have a repeated set of those

Reply:
I pushed up a change before I saw this. Let me know what you think